app/Resources/views/legislatives/platform.html.twig

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<!--
+ ______         __  __                _            _
+|  ____|       |  \/  |              | |          | |
+| |__   _ __   | \  / | __ _ _ __ ___| |__   ___  | |
+|  __| | '_ \  | |\/| |/ _` | '__/ __| '_ \ / _ \ | |
+| |____| | | | | |  | | (_| | | | (__| | | |  __/ |_|
+|______|_| |_| |_|  |_|\__,_|_|  \___|_| |_|\___| (_)
+
+-->
+<html>
+<head>
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="msapplication-tap-highlight" content="no" />
+    <meta name="viewport" content="user-scalable=no, initial-scale=1, maximum-scale=1, minimum-scale=1, width=device-width" />
+
+    <link rel="apple-touch-icon" sizes="180x180" href="{{ asset('apple-touch-icon.png') }}?v2" />
```

Comment:
instead of hardcoded `?v2` you could use the [assets version configuration](http://symfony.com/doc/current/reference/configuration/framework.html#version).